plugins/offboard/include/plugins/offboard/offboard.h

```diff
@@ -146,13 +154,19 @@ class Offboard : public PluginBase {
      */
     void set_velocity_ned(VelocityNEDYaw velocity_ned_yaw);
 
+
     /**
      * @brief Set the velocity body coordinates and yaw angular rate.
      *
      * @param velocity_body_yawspeed Velocity and yaw angular rate `struct`.
      */
     void set_velocity_body(VelocityBodyYawspeed velocity_body_yawspeed);
 
+    // Adding set attitude method.
```

Comment:
Can we have it consistent with the rest of the documentation? 

    /**
    * @brief blah blah
    ...

Reply:
Done!